### DISCUSSION TITLE
I cant import flixel.addons.ui.FlxUIState;

### DISCUSSION THREAD

**Discussion by mattsodes1031** [HaxeFlixel/flixel#2357] [Q&A]
Im doing a project where i have to import flixel.addons.ui.FlxUIState;, but whenever i do it says Type not found for some reason. i have tried installing the addons and the ui again, but it doesn't work, does anyone know how to help me, i am in a pinch.

**Comment by dean1010** [comment]
Do you have this in your project.xml?
`<haxelib name="flixel-addons" />`
`<haxelib name="flixel-ui" />`

**Reply by TheHENOOB** [reply]
Only adding the `<haxelib name="flixel-ui" />` should work normally if the dev is only looking for the ui package
